staff/diana-moreno/precourse/calculator html/calculator-basic/index.css

```diff
@@ -0,0 +1,123 @@
+.section1 {
```

Comment:
Sorry... client is so picky with colors and design ... they wants something more funny, with more lights. 
They really want to show a brand new calculator design for next Iphone generation!

Reply:
Thanks for the advice! Changed.
![calculator-basic](https://user-images.githubusercontent.com/47218719/66260877-b357e900-e7c4-11e9-917d-f15e7d072293.png)